### DISCUSSION TITLE
Recover FORK repo

### DISCUSSION THREAD

**Discussion by bilalsheikh1** [community/community#31788] [Other Feature Feedback, Questions, & Ideas]
i deleted my fork repo. so i want to recover it urgent basis

**Comment by airtower-luna** [comment]
If you have a local copy of the repository, you can just create a new fork and push your data again.

If not, you'll have to [contact GitHub support about restoring it](https://support.github.com/contact?tags=rr-restore), us other users can't access their backups. Good luck!
